client/src/modules/purchases/create/PurchaseOrderForm.js

```diff
@@ -91,26 +91,32 @@ function PurchaseOrderFormService(Inventory, AppCache, Store, Pool, PurchaseOrde
 
     // TODO(@jniles) - what do we do about user_id?  Should we
     // modify it?  Keep a table of modifications to purchase orders?
+    // NOTE(@jniles) - Vanga would like to see who modified the purchase orders.
 
-    // add the expected number of items
-    order.items.forEach((prev) => {
-      // make the item uuids match between the previous and current records
-      const current = this.addItem();
-      current.uuid = prev.uuid;
+    // make sure we have our inventory loaded before applying the old order prices.
+    return this.ready().then(() => {
```

Comment:
This code looks good, but I wonder what happens to some of the other purchase order fields such as:
- project_id
- reference
- created_at
- update_at
- user_id
- edited
- status_id
(I may be missing where these are being updated)

Reply:
Good question.  Most of these never change, or are changed in triggers in the database.   Not one of those columns is meant to be changed in this particular editing interface that the `PurchaseOrderForm` powers (createUpdate.html).